cypress/e2e/cloud/checkout/checkoutpage.test.ts

```diff
@@ -27,135 +27,127 @@ describe('Checkout Page Works', () => {
   )
 
   it('should render the checkout page and allow for pointing and clicking', () => {
-    cy.setFeatureFlagsNoNav({
-      multiOrg: true,
-    }).then(() => {
-      const email = 'asalem@influxdata.com'
-      const limit = 10
-      const numberError = 'Please enter a value of 1 or greater'
-      const genericError = 'This is a required field'
+    const email = 'asalem@influxdata.com'
+    const limit = 10
+    const numberError = 'Please enter a value of 1 or greater'
+    const genericError = 'This is a required field'
 
-      cy.getByTestID('shouldNotify--checkbox--input').should('be.checked')
+    cy.getByTestID('shouldNotify--checkbox--input').should('be.checked')
 
-      resetInputs()
+    resetInputs()
 
-      // Click Upgrade
-      cy.getByTestID('checkout-upgrade--button').click()
+    // Click Upgrade
+    cy.getByTestID('checkout-upgrade--button').click()
 
-      // Check all errors are visible
-      cy.getByTestID('balanceThreshold--input').scrollIntoView()
-      cy.getByTestID('balanceThreshold--form-element-error').should(
-        'be.visible'
-      )
-      cy.getByTestID('balanceThreshold--form-element-error').contains(
-        genericError
-      )
-      cy.getByTestID('notifyEmail--form-element-error').should('be.visible')
-      cy.getByTestID('notifyEmail--form-element-error').contains(genericError)
-
-      // Check balance threshold specific error should exist
-      cy.getByTestID('balanceThreshold--input').clear().type('0')
-      cy.getByTestID('balanceThreshold--form-element-error').contains(
-        numberError
-      )
-
-      cy.getByTestID('notifyEmail--input').clear().type(email)
-      cy.getByTestID('balanceThreshold--input').clear().type(`${limit}`)
-
-      // Check all errors are gone
-      cy.getByTestID('balanceThreshold--form-element-error').should('not.exist')
-      cy.getByTestID('notifyEmail--form-element-error').should('not.exist')
-
-      // Uncheck Checkbox
-      cy.getByTestID('shouldNotify--checkbox').click()
-      cy.getByTestID('shouldNotify--checkbox--input').should('not.be.checked')
-
-      // Email and limit should still be present after toggling the notifications checkbox
-      cy.getByTestID('shouldNotify--checkbox').click()
-      cy.getByTestID('shouldNotify--checkbox--input').should('be.checked')
-      cy.getByTestID('notifyEmail--input').should('have.value', email)
-      cy.getByTestID('balanceThreshold--input').should('have.value', limit)
-
-      // should render US Billing Address
+    // Check all errors are visible
+    cy.getByTestID('balanceThreshold--input').scrollIntoView()
+    cy.getByTestID('balanceThreshold--form-element-error').should('be.visible')
+    cy.getByTestID('balanceThreshold--form-element-error').contains(
```

Comment:
These are changes enforced by prettier. 

The line width no longer exceeds its 80-character preference since this code block is no longer part of the promise chain from `cy.setFeatureFlagsNoNav`.